File: Data_Structures/Array/Inversion_Count/Inversion_count.c
#include <stdio.h> 
#include <stdlib.h> 

//function to count the number of inversions
int invC(int arr[], int n) 
{ 
    int inv_c = 0; 
    for (int i = 0; i < n - 1; i++) 
        for (int j = i + 1; j < n; j++) 
            if (arr[i] > arr[j]) 
                inv_c++; 
  
    return inv_c; 
} 


int main() 
{ 
    int arr[] = { 1,9,2,8,3,7}; 
    int n = sizeof(arr) / sizeof(arr[0]); 
    printf(" Number of inversions in the array are %d \n",invC(arr, n)); 
    return 0; 
} 

File: Data_Structures/Array/Inversion_Count/Inversion_Count.cs
class Inversion_Count
{
    public static int inversionCount(int[] input)
    {
        var count = 0;

        for (int i = 0; i < input.Length; i++)
        {
            for (int j = 0; j < input.Length; j++)
            {
                if (input[i] > input[j])
                {
                    count++;
                }
            }
        }

        return count;
    }
}
